noir-projects/noir-protocol-circuits/crates/blob/Nargo.toml

```diff
@@ -4,6 +4,7 @@ type = "lib"
 authors = [""]
 compiler_version = ">=0.30.0"
 
+#TODO(Miranda) CHANGE BELOW BACK (currently 0.4.2 is the only working version with noir, its not released)
 [dependencies]
-bigint = {tag = "v0.4.0", git = "https://github.com/noir-lang/noir-bignum" }
+bigint = {tag = "release-please--branches--main", git = "https://github.com/noir-lang/noir-bignum" }
```

Comment:
It's pointing to `main` without pointing to `main`, which would be horrible because it changes. I'm just waiting for 0.4.2 to be merged (the above should NOT be merged into a-p master)

Reply:
We have 0.4.2 now, so ignore the above!